packages/terra-functional-testing/src/services/wdio-visual-regression-service/VisualRegressionLauncher.js

```diff
@@ -62,8 +63,33 @@ class VisualRegressionLauncher {
   /**
    * Function to be executed after a test in Mocha.
    * @param {Object} test - test details
+   * @param {Object} context - test context
+   * @param {Object} results - results object
+   * @param {boolean} results.passed - indicates if the test passed.
    */
-  afterTest() {
+  afterTest(test, _, { passed }) {
```

Comment:
personal preference. this changes how we set `name` below:
```suggestion
  afterTest({ parent, title }, _, { passed }) {
```